JS-Algorithms/challenges.js

```diff
@@ -95,7 +123,17 @@ Since 10! === 3628800 and you sum 3 + 6 + 2 + 8 + 8 + 0 + 0
 ***** */
 
 const digitSum = (n) => {
-  // YOUR CODE HERE...
+  const factorialNumbers = [];
+
+  while (n > 0) {
+    factorialNumbers.push(n);
+    n--;
+  }
+
+  const factorialSum = factorialNumbers.reduce( (acc, cc) => BigInt(acc) * BigInt(cc));
```

Comment:
Personally, I wouldn't say I like to use reduced variable names because not always everyone knows what they mean, for example, I can infer that `acc` means `accumulator`, but I am not sure what `cc` means exactly. This could create a bit of confusion if you are not sure that all the team members know that abbreviation.